keyboards/crkbd/keymaps/dsanchezseco/keymap.c

```diff
@@ -0,0 +1,115 @@
+#include QMK_KEYBOARD_H
+#include "split_util.h"
+
+extern keymap_config_t keymap_config;
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
+
+extern uint8_t is_master;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+enum crkbd_layers {
+  _DVORAK,
+  _LOWER,
+  _RAISE,
+  _ADJUST
+};
+
+enum custom_keycodes {
+  DVORAK = SAFE_RANGE,
+  LOWER = MO(_LOWER),
+  RAISE = MO(_RAISE),
+  ESC_CTL = LCTL_T(KC_ESC),
+};
```

Comment:
This entire block/enum could be removed.   If you only have DVORAK, you don't need the keycode for it.  

For the rest:
```c
#define LOWER MO(_LOWER)
#define RAISE MO(_RAISE)
#define ESC_CTL LCTL_T(KC_ESC)
```

Reply:
Thanks!